nservicebus/scalability-and-ha/deploying-to-a-windows-failover-cluster.md

```diff
@@ -133,6 +133,8 @@ Again, try swapping the cluster back and forth, to ensure it can move freely bet
 
 ## Setting up the workers
 
+NOTE: As of Version 6 NServiceBus no longer supports running an embedded distributor. Because of that configuration sections like `MasteNodeConfig` are obsoleted and `endpointConfiguraiton.EnlistWithLegacyMSMQDistributor` should be used instead. Read more about [Upgrading a Distributor-based scaled out endpoint to V6](/samples/scaleout/distributor-upgrade/)
```

Comment:
Distributor should not be used for new deployments unless there is a need for backwards compat with 5 or previous. Not sure if WARNING is not too strong.